src/components/ListItem.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <template>
+      <v-container>
+        <v-row>
+          <v-col col="12" md="6">
+            <v-text-field
+              style="margin-bottom:20px;"
+              :value="listItem.text"
+              @change="listItem.text = $event"
+              :outlined="isOutlined"
+              @click="outline"
+              @blur="handleBlur"
+              >{{ listItem.text }}</v-text-field
+            >
+          </v-col>
+          <v-col col="12" md="6">
+            <v-select
+              style="margin-bottom:20px;"
+              :items="allItemsStates"
+              :label="listItem.state"
+              @change = "listItem.state = $event"
+              :outlined="isOutlined"
+              @click="outline"
+              @blur="handleBlur"
+            ></v-select>
+          </v-col>
+        </v-row>
+      </v-container>
+    </template>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    listItem: {
+      type: Object,
+      default: () => ({}),
+      description: 'The list item',
+    },
+  },
+  data: () => ({
+    isOutlined: false,
+  }),
+  methods: {
+    handleBlur(e) {
+      this.isOutlined = false;
+      console.log('blur', e.target);
+    },
+    outline(e) {
```

Comment:
The argument here should be descriptive. What is `e` here? An event? An element? 
Self-documenting code is easier to understand. Don't be afraid to use long names, either - they get compiled down to shorter names, so performance and size is not an issue.
(Based on the code, it's not used at all, so should probably be removed.)